tap_s3_csv/__init__.py

```diff
@@ -13,21 +18,39 @@
 REQUIRED_CONFIG_KEYS = ["start_date", "bucket", "aws_access_key_id", "aws_secret_access_key"]
 
 
-def do_discover(config):
+def do_discover(config: Dict) -> None:
+    """
+    Discovers the source by connecting to the it and collecting information about the given tables/streams,
+    it dumps the information to stdout
+    :param config: connection and streams information
+    :return: nothing
+    """
     LOGGER.info("Starting discover")
     streams = discover_streams(config)
     if not streams:
         raise Exception("No streams found")
     catalog = {"streams": streams}
-    json.dump(catalog, sys.stdout, indent=2)
+    ujson.dump(catalog, sys.stdout, indent=2)
     LOGGER.info("Finished discover")
 
 
-def stream_is_selected(mdata):
-    return mdata.get((), {}).get('selected', False)
+def stream_is_selected(meta_data: Dict) -> bool:
+    """
```

Comment:
Also, since you started refactoring and putting more nice comments, we can already agree what we want to use throughout all of our repos. There are very nice google docstrings that people usually use (and it's very easy to switch to them automatically in PyCharm https://intellij-support.jetbrains.com/hc/en-us/community/posts/115000784410-how-to-change-pycharm-default-commenting-style-for-function-) wdyt?

Reply:
I'm using the default docstring structure that PyCharm provides! Do you mean use another?

Reply:
I mean if we want to :) I like those google docstrings much more in general, wdyt?

Reply:
Sure, I don't have a preference for any docstrings, as long as there is one that is :).
I switched to it in PyCharm.